deletelistView.php

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="">
+
+
+<head>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <title>Student Planner</title>
+</head>
+
+    <body>
+        <h1 class="text-center">Student Planner</h1>
+	<br>
+        <h2 class="text-center">List View</h2>
+	<br>
+
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#createList">
+  Create List
+</button>
+<?php
+require("conn.php"); 
+
+if (!empty($_POST['name'])){
+	//If the user submitted a new list request we run this code
+	$query = " 
+            INSERT INTO taskList ( 
+                listTitle,
+                boardID
+            ) VALUES ( 
+                :listTitle, 
+                :boardID
+            ) 
+        "; 
+        $query_params = array( 
+            ':listTitle' => $_POST['name'], //Insert the user's input into the database
+            ':boardID' => 1 //Placeholder until we develop the board view function
+        ); 
+        $stmt = $db->prepare($query); 
+       	$result = $stmt->execute($query_params); 
+}
+if (!empty($_POST['name'])){
+	//If the user submitted a new delete list request we run this code

```

Comment:
Some modifications
- This block (line 40-48) can be moved to `listView.php`

- Change the variable inside the `$_POST` to be different from that of the create list call (e.g. `name_to_delete`) - this change should also be made in the form(`<input name = "">`) for the delete task